notesapi/v1/tests/test_views.py

```diff
@@ -502,6 +505,24 @@ def test_search_ordering(self):
         self.assertEqual(results['rows'][1]['text'], 'Third note')
         self.assertEqual(results['rows'][2]['text'], 'First one')
 
+    @unittest.skipIf(settings.ES_DISABLED, "MySQL does not do relevance ordering")
+    def test_search_ordering_in_es(self):
+        """
+        Tests order of search results from ElasticSearch.
+
+        ElasticSearch sorting is based on the computed relevance of each hit.
+        """
+        self._create_annotation(text=u'fox of the foxes')
+        self._create_annotation(text=u'the lead fox')
+        self._create_annotation(text=u'a very long entry that contains the word fox')
+        self._create_annotation(text=u'does not mention the word')
+
+        results = self._get_search_results(text='fox')
+        self.assertEqual(results['total'], 3)
+        self.assertEqual(results['rows'][0]['text'], 'fox of the foxes')
+        self.assertEqual(results['rows'][1]['text'], 'the lead fox')
+        self.assertEqual(results['rows'][2]['text'], 'a very long entry that contains the word fox')
```

Comment:
are you sure they aren't being returned in the order they were created, rather than by relevance?